src/cljs/rems/user.cljs

```diff
@@ -43,7 +43,7 @@
           (for [[k v] other-attributes]
             (let [title (or (localized (get-in extra-attributes [(name k) :name]))
                             k)]
-              [info-field title v {:inline? true}])))))
+              [info-field title (str v) {:inline? true}])))))
```

Comment:
What sort of values do we get here? Should we build a bit smarter printing here?

Reply:
![Ohne Titel 2e](https://user-images.githubusercontent.com/24792629/209098348-9af9e364-b6ec-47bd-8441-49c410a988e1.png)
The attached screenshot was created with the suggested string conversion. It shows _email_verified_ as a boolean value, resulting in an empty form field before, while voperson_external_affilition is an array, resulting in the user profile page not being displayed in the browser at all.

Reply:
Thanks, I can add a rudimentary checkbox and array support for this (for example separated by `,`).